#http://docs.ros.org/en/api/sensor_msgs/html/msg/NavSatFix.html
float64 latitude
float64 longitude
float64 altitude
float64[9] position_covariance
uint8 position_covariance_type
---
bool state